src/spdx_tools/spdx/parser/jsonlikedict/relationship_parser.py

```diff
@@ -52,9 +52,6 @@ def parse_all_relationships(self, input_doc_dict: Dict) -> List[Relationship]:
         )
 
         package_dicts: List[Dict] = input_doc_dict.get("packages", [])
-        existing_relationships_without_comments: List[Relationship] = self.get_all_relationships_without_comments(
```

Comment:
I'm a bit confused by the removal of this code.  The semantics will be different between this and previous call right?  This one would include the additional relationships found from the relationships.extend call above.  To preserve the previous behavior, it seems that you would instead want to simply retrieve the set identified by parser_field_or_log_error, and then extend relationships with that set.  But then you could also update the existing_relationships_without_comments from that new set as well.  The question is whether you are purposefully changing the semantics here or not.